### PR TITLE
Add Tiny Yolov2 samples

### PR DESCRIPTION
**Pull request by Honry**
Supports both 'NCHW' and 'NHWC' layouts

Fixed https://github.com/webmachinelearning/webnn-samples/issues/8

### CODE REVIEW

**Comment by Honry** [comment]
@huningxin, PTAL, thanks! 

Here's the preview: https://honry.github.io/webnn-samples/object_detection/, which will be available once https://github.com/webmachinelearning/test-data/pull/6 is landed.

File: object_detection/tiny_yolov2_nchw.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        {type: 'float32', dimensions: this.inputOptions.inputDimensions});
+
+    const mulScale = this.builder_.constant({type: 'float32',
+      dimensions: [1]}, new Float32Array([0.003921568859368563]));
+    const addBias = this.builder_.constant({type: 'float32',
+      dimensions: [3, 1, 1]}, new Float32Array([0, 0, 0]));
+
+    const mul = this.builder_.mul(image, mulScale);
+    const add = this.builder_.add(mul, addBias);
+    const loop0 = await this.buildLeakyRelu_(add, '');
```
Comment (huningxin) [inline object_detection/tiny_yolov2_nchw.js:81]: Please name the layer as `conv0` and similar for following ones.

**Review by huningxin — COMMENTED** [review]
Thanks @Honry . It is a great sample. I left some comments, please take a look.

**Comment by Honry** [comment]
@huningxin, thanks for your review! I've addressed your comments, PTAL again, thanks!

**Review by huningxin — APPROVED** [review]
Looks good. Thanks @Honry !

**Pull request merged**